website/app/components/Head.tsx

```diff
@@ -0,0 +1,39 @@
+import { Helmet } from 'react-helmet';
+import { DocumentationLoader } from '~/loaders/documentation.server';
+import codeHikeStyles from '@code-hike/mdx/dist/index.css';
+
+export const Head = ({ data }: { data: DocumentationLoader }) => {
+  const favicon = getFavicon({ data });
+
+  return (
+    //@ts-ignore
+    <Helmet>
+      <link rel="icon" href={favicon} />
+      {data.config.experimentalMath && (
+        <link
+          rel="stylesheet"
+          href="https://cdn.jsdelivr.net/npm/katex@0.15.0/dist/katex.min.css"
+        />
+      )}
+      {data.config.experimentalCodehike && <link rel="stylesheet" href={codeHikeStyles} />}
+    </Helmet>
+  );
+};
+
+const getFavicon = ({ data }: { data: DocumentationLoader }) => {
+  let favicon = new URL('https://docs.page/favicon.ico?v=2');
+
+  const logoConfig = data.config.logo || data.config.logoDark;
+
+  if (logoConfig) {
+    if (logoConfig.startsWith('http')) {
+      favicon = new URL(logoConfig);
+    } else if (logoConfig.startsWith('/')) {
+      favicon = new URL(
+        `https://raw.githubusercontent.com/${data.owner}/${data.repo}/${data.source.ref}/docs${logoConfig}`,
```

Comment:
`data.source.ref` should be wrapped in `encodeURIComponent` see https://github.com/invertase/docs.page/pull/189